terracumber_config/tf_files/SUSEManager-5.0-SLE-update-NUE.tf

```diff
@@ -0,0 +1,219 @@
+}
+
+variable "CUCUMBER_BRANCH" {
+  type = string
+  default = "Manager-5.0"
+}
+
+variable "CUCUMBER_RESULTS" {
+  type = string
+  default = "/root/spacewalk/testsuite"
+}
+
+variable "MAIL_SUBJECT" {
+  type = string
+  default = "Results 5.0 SLE Update $status: $tests scenarios ($failures failed, $errors errors, $skipped skipped, $passed passed)"
+}
+
+variable "MAIL_TEMPLATE" {
+  type = string
+  default = "../mail_templates/mail-template-jenkins.txt"
+}
+
+variable "MAIL_SUBJECT_ENV_FAIL" {
+  type = string
+  default = "Results 5.0 SLE Update: Environment setup failed"
+}
+
+variable "MAIL_TEMPLATE_ENV_FAIL" {
+  type = string
+  default = "../mail_templates/mail-template-jenkins-env-fail.txt"
+}
+
+variable "MAIL_FROM" {
+  type = string
+  default = "galaxy-noise@suse.de"
+}
+
+variable "MAIL_TO" {
+  type = string
+  default = "galaxy-noise@suse.de"
+}
+
+// sumaform specific variables
+variable "SCC_USER" {
+  type = string
+}
+
+variable "SCC_PASSWORD" {
+  type = string
+}
+
+variable "GIT_USER" {
+  type = string
+  default = null // Not needed for master, as it is public
+}
+
+variable "GIT_PASSWORD" {
+  type = string
+  default = null // Not needed for master, as it is public
+}
+
+terraform {
+  required_version = "1.0.10"
+  required_providers {
+    libvirt = {
+      source = "dmacvicar/libvirt"
+      version = "0.6.3"
+    }
+  }
+}
+
+provider "libvirt" {
+  uri = "qemu+tcp://suma-08.mgr.suse.de/system"
+}
+
+module "base" {
+  source            = "./modules/base"
+
+  cc_username       = var.SCC_USER
+  cc_password       = var.SCC_PASSWORD
+  product_version   = "5.0-released"
+  name_prefix       = "suma-su-50-"
+  use_avahi         = false
+  domain            = "mgr.suse.de"
+  images            = [ "sles15sp6o", "opensuse155o", "slemicro55o" ]
+
+  mirror            = "minima-mirror-ci-bv.mgr.suse.de"
+  use_mirror_images = true
+
+  testsuite         = true
+
+  provider_settings = {
+    pool        = "ssd"
+    bridge      = "br0"
+  }
+}
+
+module "server_containerized" {
+  source             = "./modules/server_containerized"
+  base_configuration = module.base.configuration
+  name               = "server"
+  image              = "slemicro55o"
+  provider_settings = {
+    mac                = "aa:b2:93:01:03:9c"
+    memory             = 16384
+    vcpu               = 4
+    data_pool          = "ssd"
+  }
+
+  main_disk_size        = 1500
+  runtime               = "podman"
+  // Temporary workaround to see if we pass proxy stage. Also needs to be updated on next MU
+  container_repository  = "registry.suse.de/suse/maintenance/35237/suse_sle-15-sp6_update_products_manager50_update_containerfile"
+  container_tag         = "latest"
+  server_mounted_mirror = "minima-mirror-ci-bv.mgr.suse.de"
+
+  auto_accept                    = false
+  monitored                      = true
+  disable_firewall               = false
+  allow_postgres_connections     = false
```

Comment:
Do we really need to set to false all these variables?
if so, maybe we can consider change the default values of them.

Reply:
I can double check the default value in sumaform. Overall in BV / sle maintenance update pipeline we are exhaustive on those parameters. I suppose it's to avoid been impacted by default value change in sumaform.

Reply:
I'd consider QE squad the main user of this tool, and so, we better align the default values with our default use cases (CI and BV)

Reply:
I checked on sumaform side, we need those values. Changing the default is a bit risky, we don't know how other project are using sumaform. It also helps to see quickly without opening sumaform how the server is set. I would like to keep those value as is. We maybe think about changing the default if we create a template + variable files. I tried to remove all the useless parameter ( like server_configuration ) that are already not needed. Let's see for the default.

Reply:
Ok, if you both prefer setting all the variables in these environments.

📢 Still, out of scope of this PR, but I think that QE own the main need of sumaform, and we must adapt the default values to our default needs.